### PR TITLE
FloorsAndRoomCard: Change creation of title for RollershutterCard

### PR DESCRIPTION
**Pull request by Rosi2143**
uiSemantics is based on points of equipments to this point is used to create the title instead of the equipment itself

Signed-off-by: Rosi2143 Schrott.Micha@web.de

### CODE REVIEW

File: mainWidget/main_widget_FloorsAndRooms.yaml [hunk 1]
```diff
@@ -87,8 +87,8 @@ slots:
                                       groupItem: =loop.equipmentItem.name
                                       RollerItem: =loop.shutterItem.name
                                       RollerMode: = loop.modeItem.name
-                                      Title: "=loop.equipmentItem.metadata.uiSemantics ? loop.equipmentItem.metadata.uiSemantics.config.equipment + loop.equipmentItem.metadata.uiSemantics.config.preposition + loop.equipmentItem.metadata.uiSemantics.config.location : loop.equipmentItem.label"
-                                      visible: '=!vars.objVar ? false : !vars.objVar.selectThing ? false : (vars.objVar.selectThing=="Rollers") ? true : false'
+                                      Title: "=loop.shutterItem.metadata.uiSemantics ? loop.shutterItem.metadata.uiSemantics.config.equipment + loop.shutterItem.metadata.uiSemantics.config.preposition + loop.shutterItem.metadata.uiSemantics.config.location : loop.shutterItem.label"
+                                      visible: '=!vars.objVar ? false : !vars.objVar.selectThing ? true : (vars.objVar.selectThing=="Rollers") ? true : false'
```
Comment (hmerk) [inline mainWidget/main_widget_FloorsAndRooms.yaml]: What would be the benefit here ? Don't see the need for changing from equipment item to shutter item.
And please do not revert changes I have made on purpose without asking.
The reason for changing visibility was performance issues. This was also included in the comment for that change.

Comment (Rosi2143) [reply mainWidget/main_widget_FloorsAndRooms.yaml]: So finally I got it right - I shouldn't work late at night :-(

**Comment by hmerk** [comment]
Still not answered my question. What is the benefit of changing the label creation?

**Comment by Rosi2143** [comment]
As I didn't find any rule how to create the uiSemantic elements I only created these for items (points belonging to an equipment) - not for the equipment itself. This caused the title to be just the name of the equipment.
Now I also added the uiSemantic tags to equipment items and it works without this change.

**Pull request closed without merging**